lib/create-environment-api.js

```diff
@@ -924,6 +973,8 @@ export default function createEnvironmentApi ({
     getUiExtensions: getUiExtensions,
     createUiExtension: createUiExtension,
     createUiExtensionWithId: createUiExtensionWithId,
+    getAppInstallation: getAppInstallation,
```

Comment:
So we're missing `createAppInstallation(appDefinitionId, parameters)` method

Reply:
@FLoppix